src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -393,7 +393,7 @@ export default Vue.extend({
         this.publishedText = this.data.publishedText
       }
 
-      if (typeof (this.data.publishedText) !== 'undefined' && this.data.publishedText !== null && !this.isLive) {
+      if ('publishedText' in this.data && !!this.data.publishedText && !this.isLive) {
```

Comment:
This change can be simplified to this:
```suggestion
      if (this.data.publishedText && !this.isLive) {
```